Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM node:6 AS builder
+
+ENV ENV_NODE=production
+
+RUN groupadd --gid 1100 lisk && \
```

Comment:
I can't find any reference for using `1100` gid specific to distro. Why we are hardcoding the group id. What if some dependent docker container use the same group id? 

Would it not be nice to add group only with name and then get its id afterwards. This may be helpful `getent group lisk`

Reply:
The GID is not distribution specific. We chose `1100` arbitrarily.

> What if some dependent docker container use the same group id?
That would be no problem in a different container. I you mean a container that mounts a volume from this one then that'd allow having the "same" lisk user (same UID and GID).